4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-general-editor-duplicate-projects-straight-from-the-project-manager.md

---
type: entry
section: general
subsection: editor
rank: 8
importance: 4
anchor: duplicate-projects-straight-from-the-project-manager
title: Duplicate projects straight from the Project manager
text: |

>   There's now a dedicated "Duplicate" button in the Project manager, making it easy to backup an existing project, or to fork a new one.

Reviewer: ```suggestion
  There’s now a dedicated "Duplicate" button in the Project manager, making it easy to back up an existing project, or to fork a new one.
```
"backup" is a noun, the result of the verb phrase "back up"

>   There’s now a dedicated "Duplicate" button in the Project manager, making it easy to backup an existing project, or to fork a new one.

Reviewer: ```suggestion
  There’s now a dedicated "Duplicate" button in the Project manager, making it easy to back up an existing project, or to fork a new one.
```
You skipped this one, it's "back up" for the action, "backup" for the item (you perform a "backup", but you "back up")

Author: Oh! I corrected the one in the next paragraph. I think you just missed that one. :P

Reviewer: No that's the same line: https://github.com/godotengine/godot-website/pull/1138#discussion_r2267403546 :P (as in you missed this one)

>   There’s now a dedicated "Duplicate" button in the Project manager, making it easy to backup an existing project, or to fork a new one.

Reviewer 2: ```suggestion
  There’s now a dedicated "Duplicate" button in the Project Manager, making it easy to backup an existing project, or to fork a new one.
```
As I recall, "Project Manager" is the proper name of the whole window/app that lets you select a project

>   We also figured that it would be a great tool to prevent losses if you update the engine version during development. If we detect that irreversible changes would occur due to the update, we'll ask you if you want to backup your project first.

Reviewer: ```suggestion
  We also figured that it would be a great tool to prevent losses if you update the engine version during development. If we detect that irreversible changes would occur due to the update, we’ll ask you if you want to backup your project first.
```

>   We also figured that it would be a great tool to prevent losses if you update the engine version during development. If we detect that irreversible changes would occur due to the update, we’ll ask you if you want to back up your project first.

Reviewer: ```suggestion
  We also figured that it would be a great tool to prevent losses if you update the engine version during development. If we detect that irreversible changes would occur due to the update, we’ll ask if you want to back up your project first.
```
Redundant and slows down the flow IMO
